Example/novoda-constraints/AppDelegate.swift

```diff
@@ -1,46 +1,15 @@
-//
-//  AppDelegate.swift
-//  novoda-constraints
-//
-//  Created by Giuseppe Basile on 05/09/2018.
-//  Copyright (c) 2018 Giuseppe Basile. All rights reserved.
-//
-
 import UIKit
 
 @UIApplicationMain
 class AppDelegate: UIResponder, UIApplicationDelegate {
 
     var window: UIWindow?
```

Comment:
you can directly create the windows here and avoid all the optionalities

```suggestion
    let window = UIWindow(frame: UIScreen.main.bounds)
```